extras/check_code_style.sh

```diff
@@ -232,7 +239,13 @@ for f in $file_names; do
     *.py )
       echo "Check PEP8 on file $f:"
 
-      if ! pep8_result=`pep8 --first --ignore=$PEP8_IGNORES $f` ; then
+      if [[ $f =~ $EXAMPLE_DIRS ]]; then
```

Comment:
Isn't there a comment flag within the file to mark sections that should be ignored by pep8? I suggest it only to avoid a proliferation of cases here where ignores should be changed.